packages/core/src/storage/chatStore/SimpleChatStore.ts

```diff
@@ -0,0 +1,92 @@
+import { ChatMessage } from "../../llm";
+import { BaseChatStore } from "./types";
+
+/**
+ * Simple chat store.
+ */
+export class SimpleChatStore implements BaseChatStore {
```

Comment:
Not a blocker: So this class feels like a bit unnecessary in JS. Not sure it adds anything particularly useful over just the underlying data structure.